scripts/gcloud_adapter.py

```diff
@@ -14,13 +14,51 @@
 
 """Module with GCloud-related functions."""
 
+import json
 import os
 import subprocess
+import sys
 
 GCLOUD_PATH = os.path.join(
     '..', 'oppia_tools', 'google-cloud-sdk-251.0.0', 'google-cloud-sdk',
     'bin', 'gcloud')
 
+CURR_DIR = os.path.abspath(os.getcwd())
+OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
+THIRD_PARTY_DIR = os.path.join(CURR_DIR, 'third_party')
+
+DIRS_TO_ADD_TO_SYS_PATH = [
```

Comment:
Why do you need all this stuff? Could you just use the ones that are needed by the remote API stub?

Reply:
Checking this once and removing the extra ones.

Reply:
I was using a wrong path to import remote api shell earlier and so these imports were required. I missed removing them. Updated now.